src/app/pages/Kafka/ConnectedMetrics.tsx

```diff
@@ -32,13 +38,13 @@ export const ConnectedMetrics: VoidFunctionComponent<ConnectedMetricsProps> = ({
       return {
         ...metrics,
         ...(instanceType === "standard" ? {
-          connectionsLimit:3000,
-          connectionRateLimit:100,
-          diskSpaceLimit:1000
+          connectionsLimit: 3000,
```

Comment:
We'll need to pull this data from the API not that instances can come in different sizes.